docker/docker/src/main/java/org/arquillian/cube/docker/impl/client/CubeDockerConfigurationResolver.java

```diff
@@ -196,6 +207,30 @@ private Machine getFirstMachine(Set<Machine> machines) {
 
         return config;
     }
+    
+    private Map<String, String> resolveSystemDefaultSetup(Map<String, String> config) {
+        if (!config.containsKey(CubeDockerConfiguration.DOCKER_URI)) {
+            final String defaultUri = operatingSystem.getDefaultFamily().getServerUri();
+            URI uri = URI.create(defaultUri);
+            if (Files.exists(FileSystems.getDefault().getPath(uri.getPath()))){
+
+                DockerClient client = defaultDocker.GetDefaultDockerClient(defaultUri);
```

Comment:
Can you change method to be Java style? (Camel case)

Reply:
the `GetDefaultDockerClient` right?